src/components/Articles/Article/Article.js

```diff
@@ -207,6 +207,16 @@ export class Article extends Component {
                   )}
                 </div>
                 <div className="divider light" />
+                <div>
+                  {article.tagList
+                    ? article.tagList.map(value => (
+                        <p className="tags-values" key={value}>
+                          <span className="value-tag">{value}</span>
+                        </p>
+                    ))
+                    : 'No tags added'}
```

Comment:
If there is no tag added to the article, I don't think it is not needed to tell visitors that there is no `tag added`.

Reply:
Yes, you are right, I have already removed the message.